### ISSUE TITLE
Cross-staff grace notes do not allocate space for themselves

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#924]
When a grace note is placed on another staff other than its home staff, no extra space is allocated to the layout to fit the grace note.  Example:

<img width="1154" alt="screen shot 2018-07-30 at 1 18 52 pm" src="https://user-images.githubusercontent.com/3487289/43394402-ee33dcb0-93fa-11e8-8a0a-26a7f66392a3.png">

In each measure, all of the grace notes should occur before the first notes in the measure.  But grace notes moved to other staves are not allocated space before placing them on the staff.  Green notes are cross-staff notes, and pink notes are displayed on their home staves.

Test MEI data:

```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-07-30T13:19:06" version="2.0.0-dev-6cdb4a3">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001638392445">
                <score xml:id="score-0000000343688006">
                    <scoreDef xml:id="scoredef-0000000128560620" midi.bpm="400">
                        <staffGrp xml:id="staffgrp-0000001392108892" symbol="brace" barthru="true">
                            <staffDef xml:id="staffdef-0000001064046538" clef.shape="G" clef.line="2" n="1" lines="5">
                                <label xml:id="label-0000000394548819" />
                            </staffDef>
                            <staffDef xml:id="staffdef-0000000320987717" clef.shape="F" clef.line="4" n="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-L1F1">
                        <measure xml:id="measure-L3" n="1">
                            <staff xml:id="staff-L3F2N1" n="1">
                                <layer xml:id="layer-L3F2N1" n="1">
                                    <note xml:id="note-L4F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L3F1N1" n="2">
                                <layer xml:id="layer-L3F1N1" n="1">
                                    <note xml:id="note-L4F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L5" n="2">
                            <staff xml:id="staff-L5F2N1" n="1">
                                <layer xml:id="layer-L5F2N1" n="1">
                                    <beam xml:id="beam-L6F2-L7F2">
                                        <note xml:id="note-L6F2" type="marked" dur="8" staff="2" oct="3" pname="c" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L7F2" type="marked" dur="8" staff="2" oct="3" pname="e" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L8F2" type="marked" dur="8" oct="3" pname="g" color="hotpink" grace="unacc" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L9F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L5F1N1" n="2">
                                <layer xml:id="layer-L5F1N1" n="1">
                                    <note xml:id="note-L9F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L10" n="3">
                            <staff xml:id="staff-L10F2N1" n="1">
                                <layer xml:id="layer-L10F2N1" n="1">
                                    <note xml:id="note-L11F2" type="marked" dur="8" oct="4" pname="e" color="hotpink" grace="unacc" accid.ges="n" />
                                    <beam xml:id="beam-L12F2-L13F2">
                                        <note xml:id="note-L12F2" type="marked" dur="8" staff="2" oct="3" pname="c" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L13F2" type="marked" dur="8" staff="2" oct="3" pname="e" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L14F2" type="marked" dur="8" oct="3" pname="g" color="hotpink" grace="unacc" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L15F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L10F1N1" n="2">
                                <layer xml:id="layer-L10F1N1" n="1">
                                    <note xml:id="note-L15F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L16" n="4">
                            <staff xml:id="staff-L16F2N1" n="1">
                                <layer xml:id="layer-L16F2N1" n="1">
                                    <note xml:id="note-L17F2" type="marked" dur="8" staff="2" oct="4" pname="e" color="limegreen" grace="unacc" accid.ges="n" />
                                    <beam xml:id="beam-L18F2-L19F2">
                                        <note xml:id="note-L18F2" type="marked" dur="8" staff="2" oct="3" pname="c" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L19F2" type="marked" dur="8" staff="2" oct="3" pname="e" color="limegreen" grace="unacc" accid.ges="n" />
                                        <note xml:id="note-L20F2" type="marked" dur="8" oct="3" pname="g" color="hotpink" grace="unacc" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L21F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L16F1N1" n="2">
                                <layer xml:id="layer-L16F1N1" n="1">
                                    <note xml:id="note-L21F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L22" n="5">
                            <staff xml:id="staff-L22F2N1" n="1">
                                <layer xml:id="layer-L22F2N1" n="1">
                                    <note xml:id="note-L23F2" type="marked" dur="8" staff="2" oct="4" pname="e" color="limegreen" grace="unacc" accid.ges="n" />
                                    <note xml:id="note-L24F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L22F1N1" n="2">
                                <layer xml:id="layer-L22F1N1" n="1">
                                    <note xml:id="note-L24F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L25" right="end" n="6">
                            <staff xml:id="staff-L25F2N1" n="1">
                                <layer xml:id="layer-L25F2N1" n="1">
                                    <note xml:id="note-L26F2" type="marked" dur="8" oct="4" pname="e" color="hotpink" grace="unacc" accid.ges="n" />
                                    <note xml:id="note-L27F2" dur="1" oct="4" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L25F1N1" n="2">
                                <layer xml:id="layer-L25F1N1" n="1">
                                    <note xml:id="note-L27F1" dur="1" oct="3" pname="c" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

Related to issue https://github.com/humdrum-tools/verovio-humdrum-viewer/issues/143

**Comment by cividini** [comment]
The same problem in cross-hand notation ("mano sinistra"). Here an example from Mozart’s piano sonata in A major K. 331, second movement:
![image](https://user-images.githubusercontent.com/45197272/87405760-28f8ad00-c5c0-11ea-89f5-019ec48a3d64.png)
In Verovio (development version 3.0.0-dev-e4b3fc2) the grace notes from staff="2" in red have no place in the horizontal plane and collide with the main notes:
![image](https://user-images.githubusercontent.com/45197272/87405683-12eaec80-c5c0-11ea-8561-8c1bc543b4e7.png)
```<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei meiversion="4.0.0" xmlns="http://www.music-encoding.org/ns/mei">
	<meiHead>
		<fileDesc>
			<titleStmt>
				<title>example cross staff grace note collision</title>
			</titleStmt>
			<pubStmt/>
		</fileDesc>
	</meiHead>
	<music>
		<body>
			<mdiv>
				<score>
					<scoreDef key.mode="major" key.pname="a" key.sig="3s" meter.count="3" meter.unit="4" xml:id="scoreDef_02">
						<staffGrp bar.thru="true" symbol="brace" xml:id="staffGrp_02">
							<staffDef clef.line="2" clef.shape="G" key.sig="2s" lines="5" n="1" xml:id="staffDef_P1_sc02"/>
							<staffDef clef.line="4" clef.shape="F" key.sig="2s" lines="5" n="2" xml:id="staffDef_P2_sc02"/>
						</staffGrp>
					</scoreDef>
					<section>
						<measure n="8" xml:id="m8_k331_002_02">
							<staff n="1" xml:id="staff_12690">
								<layer n="1" xml:id="layer_12696">
									<beam xml:id="beam_12702">
										<chord dur="8" stem.dir="down" tstamp="1" xml:id="chord_12708">
											<note oct="4" pname="b" xml:id="note_12714"/>
											<note oct="4" pname="g" xml:id="note_12720">
												<accid accid="s" xml:id="accid_12726"/>
											</note>
										</chord>
										<note dur="8" oct="4" pname="e" stem.dir="down" tstamp="1.5" xml:id="note_12732"/>
										<chord dur="8" stem.dir="down" tstamp="2" xml:id="chord_12738">
											<note oct="5" pname="d" xml:id="note_12744"/>
											<note oct="4" pname="f" xml:id="note_12750">
												<accid accid.ges="s" xml:id="accid_12756"/>
											</note>
										</chord>
										<chord dur="8" stem.dir="down" tstamp="2.5" xml:id="chord_12762">
											<note oct="5" pname="c" xml:id="note_12768">
												<accid accid.ges="s" xml:id="accid_12774"/>
											</note>
											<note oct="4" pname="e" xml:id="note_12780"/>
										</chord>
										<chord dur="8" stem.dir="down" tstamp="3" xml:id="chord_12786">
											<note oct="4" pname="b" xml:id="note_12792"/>
											<note oct="4" pname="d" xml:id="note_12798"/>
										</chord>
										<chord dur="8" stem.dir="down" tstamp="3.5" xml:id="chord_12804">
											<note oct="4" pname="a" xml:id="note_12810"/>
											<note oct="4" pname="c" xml:id="note_12816">
												<accid accid.ges="s" xml:id="accid_12822"/>
											</note>
										</chord>
									</beam>
								</layer>
							</staff>
							<staff n="2" xml:id="staff_12828">
								<layer n="1" xml:id="layer_12834">
									<note dur="4" oct="3" pname="e" tstamp="1" xml:id="note_12840"/>
									<beam xml:id="beam_12846">
										<note dur="8" oct="6" pname="d" staff="1" stem.dir="up" tstamp="2" xml:id="note_12852"/>
										<note dur="8" oct="6" pname="c" staff="1" stem.dir="up" tstamp="2.5" xml:id="note_12858">
											<accid accid.ges="s" xml:id="accid_12864"/>
										</note>
										<note dur="8" oct="5" pname="b" staff="1" stem.dir="up" tstamp="3" xml:id="note_12870"/>
										<note dur="8" oct="5" pname="a" staff="1" stem.dir="up" tstamp="3.5" xml:id="note_12876"/>
									</beam>
								</layer>
							</staff>
							<dir place="above" staff="1" startid="#note_12852" xml:id="dir_12888">
								<rend fontstyle="normal" halign="center" xml:id="rend_12894">m.&#8198;s.</rend>
							</dir>
							<slur curvedir="below" endid="#note_12732" staff="1" startid="#chord_12708" xml:id="slur_12912"/>
							<slur curvedir="below" endid="#chord_12804" staff="1" startid="#chord_12738" xml:id="slur_12924"/>
							<slur curvedir="above" endid="#note_12876" staff="1" startid="#note_12852" xml:id="slur_12930"/>
						</measure>
						<measure n="9" xml:id="m9_k331_002_02">
							<staff n="1" xml:id="staff_12942">
								<layer n="1" xml:id="layer_12948">
									<beam xml:id="beam_12954">
										<chord dur="8" stem.dir="down" tstamp="1" xml:id="chord_12960">
											<note oct="4" pname="g" xml:id="note_12966">
												<accid accid="s" xml:id="accid_12972"/>
											</note>
											<note oct="3" pname="b" xml:id="note_12978"/>
										</chord>
										<note dur="8" oct="4" pname="e" stem.dir="down" tstamp="1.5" xml:id="note_12984"/>
										<chord dur="8" stem.dir="down" tstamp="2" xml:id="chord_12990">
											<note oct="4" pname="a" xml:id="note_12996"/>
											<note oct="4" pname="c" xml:id="note_13002">
												<accid accid.ges="s" xml:id="accid_13008"/>
											</note>
										</chord>
										<note dur="8" oct="4" pname="e" stem.dir="down" tstamp="2.5" xml:id="note_13014"/>
										<chord dur="8" stem.dir="down" tstamp="3" xml:id="chord_13020">
											<note oct="5" pname="c" xml:id="note_13026">
												<accid accid.ges="s" xml:id="accid_13032"/>
											</note>
											<note oct="4" pname="a" xml:id="note_13038"/>
										</chord>
										<note dur="8" oct="4" pname="e" stem.dir="down" tstamp="3.5" xml:id="note_13044"/>
									</beam>
								</layer>
							</staff>
							<staff n="2" xml:id="staff_13050">
								<layer n="1" xml:id="layer_13056">
									<note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13062">
										<accid accid="s" xml:id="accid_13068"/>
									</note>
									<note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="1" xml:id="note_13074"/>
									<note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13080">
										<accid accid.ges="s" xml:id="accid_13086"/>
									</note>
									<note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="2" xml:id="note_13092"/>
									<note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13098">
										<accid accid.ges="s" xml:id="accid_13104"/>
									</note>
									<note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="3" xml:id="note_13110"/>
								</layer>
							</staff>
							<slur curvedir="below" endid="#note_13044" staff="1" startid="#chord_12960" xml:id="slur_13116"/>
							<slur curvedir="above" endid="#note_13074" staff="1" startid="#note_13062" xml:id="slur_13122"/>
							<slur curvedir="above" endid="#note_13092" staff="1" startid="#note_13080" xml:id="slur_13128"/>
							<slur curvedir="above" endid="#note_13110" staff="1" startid="#note_13098" xml:id="slur_13134"/>
						</measure>
					</section>
				</score>
			</mdiv>
		</body>
	</music>
</mei>```

**Comment by cividini** [comment]
The same issue in #885.

**Comment by cividini** [comment]
The key to the solution of this issue has been already suggested by @craigsapp in issue https://github.com/rism-ch/verovio/issues/1562#issuecomment-659547436: if the cross-staff notation is treated as an additional layer to the away staff, the collision is solved. Here the measure from Mozart's K. 331 with grace notes, the first with `<staff n="2">`, the second with `<layer n="2">` (Verovio Version 3.1.0-dev-b649cbf):
![cross_hand_grace_note_collision_layer_staff](https://user-images.githubusercontent.com/45197272/98525369-a5eb1900-2278-11eb-8e7f-486d146022a6.PNG)

MEI-Code:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei meiversion="4.0.0" xmlns="http://www.music-encoding.org/ns/mei">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title>example cross staff grace note collision</title>
   </titleStmt>
   <pubStmt/>
  </fileDesc>
 </meiHead>
 <music>
  <body>
   <mdiv>
    <score>
     <scoreDef key.mode="major" key.pname="a" key.sig="3s" meter.count="3" meter.unit="4" xml:id="scoreDef_02">
      <staffGrp bar.thru="true" symbol="brace" xml:id="staffGrp_02">
       <staffDef clef.line="2" clef.shape="G" key.sig="2s" lines="5" n="1" xml:id="staffDef_P1_sc02"/>
       <staffDef clef.line="4" clef.shape="F" key.sig="2s" lines="5" n="2" xml:id="staffDef_P2_sc02"/>
      </staffGrp>
     </scoreDef>
     <section>
      <measure n="9" xml:id="m9_k331_002_02">
       <staff n="1" xml:id="staff_12942">
        <layer n="1" xml:id="layer_12948">
         <beam xml:id="beam_12954">
          <chord dur="8" stem.dir="down" tstamp="1" xml:id="chord_12960">
           <note oct="4" pname="g" xml:id="note_12966">
            <accid accid="s" xml:id="accid_12972"/>
           </note>
           <note oct="3" pname="b" xml:id="note_12978"/>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="1.5" xml:id="note_12984"/>
          <chord dur="8" stem.dir="down" tstamp="2" xml:id="chord_12990">
           <note oct="4" pname="a" xml:id="note_12996"/>
           <note oct="4" pname="c" xml:id="note_13002">
            <accid accid.ges="s" xml:id="accid_13008"/>
           </note>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="2.5" xml:id="note_13014"/>
          <chord dur="8" stem.dir="down" tstamp="3" xml:id="chord_13020">
           <note oct="5" pname="c" xml:id="note_13026">
            <accid accid.ges="s" xml:id="accid_13032"/>
           </note>
           <note oct="4" pname="a" xml:id="note_13038"/>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="3.5" xml:id="note_13044"/>
         </beam>
        </layer>
       </staff>
       <staff n="2" xml:id="staff_13050">
        <layer n="1" xml:id="layer_13056">
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13062">
          <accid accid="s" xml:id="accid_13068"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="1" xml:id="note_13074"/>
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13080">
          <accid accid.ges="s" xml:id="accid_13086"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="2" xml:id="note_13092"/>
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13098">
          <accid accid.ges="s" xml:id="accid_13104"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="3" xml:id="note_13110"/>
        </layer>
       </staff>
       <slur curvedir="below" endid="#note_13044" staff="1" startid="#chord_12960" xml:id="slur_13116"/>
       <slur curvedir="above" endid="#note_13074" staff="1" startid="#note_13062" xml:id="slur_13122"/>
       <slur curvedir="above" endid="#note_13092" staff="1" startid="#note_13080" xml:id="slur_13128"/>
       <slur curvedir="above" endid="#note_13110" staff="1" startid="#note_13098" xml:id="slur_13134"/>
      </measure>
      <measure n="9" xml:id="m9_k331_002_02a">
       <staff n="1" xml:id="staff_12942a">
        <layer n="1" xml:id="layer_12948a">
         <beam xml:id="beam_12954a">
          <chord dur="8" stem.dir="down" tstamp="1" xml:id="chord_12960a">
           <note oct="4" pname="g" xml:id="note_12966a">
            <accid accid="s" xml:id="accid_12972a"/>
           </note>
           <note oct="3" pname="b" xml:id="note_12978a"/>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="1.5" xml:id="note_12984a"/>
          <chord dur="8" stem.dir="down" tstamp="2" xml:id="chord_12990a">
           <note oct="4" pname="a" xml:id="note_12996a"/>
           <note oct="4" pname="c" xml:id="note_13002a">
            <accid accid.ges="s" xml:id="accid_13008a"/>
           </note>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="2.5" xml:id="note_13014a"/>
          <chord dur="8" stem.dir="down" tstamp="3" xml:id="chord_13020a">
           <note oct="5" pname="c" xml:id="note_13026a">
            <accid accid.ges="s" xml:id="accid_13032a"/>
           </note>
           <note oct="4" pname="a" xml:id="note_13038a"/>
          </chord>
          <note dur="8" oct="4" pname="e" stem.dir="down" tstamp="3.5" xml:id="note_13044a"/>
         </beam>
        </layer>
        <layer n="2" xml:id="layer_13056a">
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13062a">
          <accid accid="s" xml:id="accid_13068a"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="1" xml:id="note_13074a"/>
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13080a">
          <accid accid.ges="s" xml:id="accid_13086a"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="2" xml:id="note_13092a"/>
         <note color="red" dur="16" grace="unknown" oct="6" pname="d" staff="1" stem.dir="up" xml:id="note_13098a">
          <accid accid.ges="s" xml:id="accid_13104a"/>
         </note>
         <note dur="4" oct="6" pname="e" staff="1" stem.dir="up" tstamp="3" xml:id="note_13110a"/>
        </layer>
       </staff>
       <staff n="2" xml:id="staff_13050a"/>
       <slur curvedir="below" endid="#note_13044a" staff="1" startid="#chord_12960a" xml:id="slur_13116a"/>
       <slur curvedir="above" endid="#note_13074a" staff="1" startid="#note_13062a" xml:id="slur_13122a"/>
       <slur curvedir="above" endid="#note_13092a" staff="1" startid="#note_13080a" xml:id="slur_13128a"/>
       <slur curvedir="above" endid="#note_13110a" staff="1" startid="#note_13098a" xml:id="slur_13134a"/>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>

```
See especially issue #1607 .
Similar solution in issue #1566 .
Other related collisions in cross-staff notation: #1792 , #1799 , #1498 .

**Comment by lpugin** [comment]
Fixed with 7c729e131850c20aa1c757ac4a7083df0abdebde

![image](https://user-images.githubusercontent.com/689412/100660712-b449c380-3352-11eb-8dc2-17d5f40b5ed6.png)

![image](https://user-images.githubusercontent.com/689412/100660765-c461a300-3352-11eb-8129-4a7c7a8dcf96.png)
